tests/src/test/java/org/neo4j/graphalgo/impl/ShortestPathTest_152.java

```diff
@@ -125,19 +125,18 @@ public void testDirectRoadOrRail() {
         // see https://github.com/neo4j-contrib/neo4j-graph-algorithms/issues/722 and fix test afterwards
         verify(mock, times(1)).accept(eq(0.0));
         verify(mock, times(1)).accept(eq(50.0));
-        verify(mock, times(1)).accept(eq(90.0));
-        verify(mock, times(1)).accept(eq(120.0));
-        verify(mock, times(1)).accept(eq(140.0));
+        verify(mock, times(1)).accept(eq(80.0));
```

Comment:
@knutwalker we have *NO* idea why this commit changed the expected result to the correct one. Maybe you can explain?

Reply:
@s1ck beats me, the test in question doesn't even call the procedure… Do you maybe have a newer version of 3.5? IIRC, the test correctness depends on whether we get the correct relationship first from the kernel. Changes there could change the outcome of this test.